src/components/transaction/DataFormat.vue

```diff
@@ -21,50 +22,63 @@ export default defineComponent({
         },
     },
     setup(props) {
-        const { actionName, actionData, useColor } = toRefs(props);
-        const dataBox = ref(null);
+        const { actionName, actionData } = toRefs(props);
+        const dataBox = ref<null | HTMLElement>(null);
         const showOverflow = ref(false);
-        const maxHeight = ref(57);
-        const switchHeight = ref(20);
         const isOverflowing = ref(false);
         const transferData = computed(() => actionData.value as TransferData);
         const clientHeight = computed(() => dataBox.value?.clientHeight ?? 0);
+        let currentData = ref<string | unknown>(null);
+        const maxHeight = 57; // the maximum row height
+        const switchHeight = 20;
+        const maxHeightStyle = computed(() =>
```

Comment:
this doesn't need to be computed since the values don't change or need to be reactive. optionally you could remove the `switchHeight` variable which isn't used elsewhere and just have `const maxHeightStyle = 37px // comment` whichever you prefer for readability.
```suggestion
        const maxHeightStyle = `calc(${maxHeight}px - ${switchHeight}px)`;
```